src/visitors/pages/Signup.js

```diff
@@ -165,6 +163,10 @@ export default class Main extends Component {
   createVisitor = (e) => {
     e.preventDefault();
 
+    if (!this.state.phone || !this.state.email) {
```

Comment:
This needs to be an `&&`

Reply:
I'm 98.3% sure this is correct.

`![value] || ![value]` is the same as `!([value] && [value])`

There's a boolean law thing about it but I can't remember it's name 🤷🏽‍♀️

Reply:
Added tests for this too